packages/@adobe/spectrum-css-temp/components/radio/skin.css

```diff
@@ -189,3 +189,45 @@ governing permissions and limitations under the License.
     }
   }
 }
+
+@media (forced-colors: active) {
+  .spectrum-Radio {
+    forced-color-adjust: none;
+    --spectrum-radio-circle-border-color-error-key-focus: ButtonText;
+    --spectrum-radio-circle-border-color-error-selected-key-focus: Highlight;
+    --spectrum-radio-circle-border-color-key-focus: Highlight;
+    --spectrum-radio-emphasized-circle-background-color: ButtonFace;
+    --spectrum-radio-emphasized-circle-border-color-disabled: GrayText;
+    --spectrum-radio-emphasized-circle-border-color-down: Highlight;
```

Comment:
Looks like the XD file states that color-down on a non-selected radio should match the default color of a non-selected radio. 
<img width="341" alt="image" src="https://user-images.githubusercontent.com/9661127/175719355-0f3fde90-bbb2-410c-9e1d-3cb6fe834d79.png">  
Should this be ButtonText?

Reply:
I decided the XD was internally inconsistent and went for the approach in the "standalone? row where they use Highlight for this. Not sure it is possible (or desirable to have the down state be different depending on whether the radio button has a label or not!
 
<img width="850" alt="Screen Shot 2022-06-27 at 3 19 22 PM" src="https://user-images.githubusercontent.com/1724479/176045370-485179fb-86d8-4278-a452-3dc561d53aca.png">

Reply:
Update: Spectrum agrees that these two should be consistent aka what you have currently is fine